packages/react-core/src/next/components/Select.md

```diff
@@ -0,0 +1,80 @@
+---
+id: Select
+section: components
+subsection: menus
+cssPrefix: pf-c-select
+propComponents:
+  ['Select', 'SelectOption', 'SelectGroup', 'SelectList', 'MenuToggle', 'SelectToggleProps', 'SelectPopperProps']
+ouia: true
+---
+
+import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
+import BellIcon from '@patternfly/react-icons/dist/esm/icons/bell-icon';
+
+## Examples
+
+### Single select
+Single select dropdown menus allow your users to select a single option from a list of options. To change the name of the options, alter the ‘itemId’ property.
+
+The initial text is listed in the select box to guide what options the user can select from. To change this text, alter the ‘React.useState’ property.
+
+You can also add more options by adding `<SelectOption>`. You can follow the format of the preexisting options to ensure proper application.
+```ts file="./SelectBasic.tsx"
+
+```
+
```

Comment:
```suggestion

### Option variations

Showcases different option variants and customizations that are commonly used in a select menu. For a more complete list, see the [Menu documentation](/components/menus/menu).

```ts file="./SelectOptionVariations.tsx"

```

```

this is missing - you may still wish to edit